tutorials/tutorial-hello-timescale.md

```diff
@@ -79,17 +97,21 @@ by typing the command below into your terminal,
 ensuring that you replace the {curly brackets} with your real
 password, hostname, and port number.
 
+>:TIP:Remember, this connection string assumes you are connecting to **Timescale Forge**. Change the user and database name as needed depending on how you're connecting.
```

Comment:
Admittedly, the image above is for Timescale Cloud. It would need to be changed out for Timescale Forge if we head in this direction.